roles/openshift_logging/README.md

```diff
@@ -160,3 +161,18 @@ Elasticsearch OPS too, if using an OPS cluster:
   need to set this
 - `openshift_logging_mux_buffer_queue_limit`: Default `[1024]` - Buffer queue limit for Mux.
 - `openshift_logging_mux_buffer_size_limit`: Default `[1m]` - Buffer chunk limit for Mux.
+- `openshift_logging_mux_file_buffer_limit`: Default `[2Gi]` per output - Mux will
+  set the value to the file buffer limit.
```

Comment:
The number of outputs is taken into consideration when this value is consumed by run.sh

https://github.com/nhosoi/origin-aggregated-logging/blob/47463c55ccf9dd9f334a8fdc21fab6f36de196c4/fluentd/run.sh#L155-L164